### PR TITLE
Jb enhancement location

### PR DESCRIPTION
**Pull request by myjyby**
Changed the based geo referencing logic.

- [x] added two tables to general db: adm0 and adm0_subunits. These are essentially copies of Natural Earth shapefiles
- [x] created a script to clean everything up and add necessary information in the locations table and in the two new tables
- [x] changed all calls to countries and country_names tables to calls to adm0_subunits and adm0 tables
- [x] make location storage changes for import module

### CODE REVIEW

File: routes/save/pad/index.js [hunk 1]
```diff
@@ -23,6 +25,8 @@ module.exports = (req, res) => {
 		;`, [ insert, uuid ])
 	} else { // UPDATE OBJECT
 		const condition = DB.pgp.as.format(` WHERE id = $1::INT;`, [ id ])
+		// req.body.sections = JSON.stringify(req.body.sections)
+		console.log(req.body.sections)
```
Comment (this-pama) [inline routes/save/pad/index.js:29]: remove console.log

File: routes/apis/contributors/json.js [hunk 2]
```diff
@@ -7,14 +7,15 @@ const fs = require('fs')
...
 
 const filter = include('routes/browse/contributors/filter')
 
 module.exports = async (req, res) => {
-	let { output, render, include_data, include_teams, include_contributions } = Object.keys(req.query)?.length ? req.query : Object.keys(req.body)?.length ? req.body : {} // req.body || {}
+	let { output, render, include_data, include_teams, include_contributions } = Object.keys(req.query)?.length ? req.query : Object.keys(req.body)?.length ? req.body : {}
 	const pw = req.session.email || null
-	const language = checklanguage(req.params?.language || req.body.language || req.session.language)
+	const language = checklanguage(req.params?.language || req.query.language || req.body.language || req.session.language)
+	console.log('language', language)
```
Comment (this-pama) [inline routes/apis/contributors/json.js:18]: some console.log =:)

**Comment by JosuaKrause** [comment]
- write setup description
- test ocean pins

**Review by JosuaKrause — APPROVED** [review]
lgtm

File: routes/browse/pads/filter.js [hunk 3]
```diff
@@ -20,14 +20,24 @@ module.exports = async (req, res) => {
...
+
 				return t.oneOrNone(`
-					SELECT iso3, name FROM country_names
-					WHERE (iso3 ILIKE $1
-						OR LOWER(name) = LOWER($1))
-						AND language = $2
+					WITH l AS (
+						SELECT COALESCE(
+							(SELECT su_a3 FROM adm0_subunits WHERE su_a3 ILIKE $1), 
+							(SELECT su_a3 FROM adm0 WHERE adm0_a3 ILIKE $1)
```
Comment (myjyby) [inline routes/browse/pads/filter.js:30]: Issue here: this should be (SELECT adm0_a3 FROM adm0…)

**Pull request merged**